Samples/PowerShell/NTA.AddCBQoSSources.ps1

```diff
@@ -0,0 +1,57 @@
+# This sample scripts demonstrates how to add enabled CBQoS sources 
+# with required Orion node and interfaces. Interfaces are discovered with NPM 
+# discovery feature.
+# You need to be logged as a user with enabled Allow Node Management Rights.
+
+
+# Connect to SWIS
+$hostname = "swis-machine-hostname"                     # Update to match your configuration
+$username = "admin"                                     # Update to match your configuration
+$password = New-Object System.Security.SecureString     # Update to match your configuration
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+$swis = Connect-Swis -host $hostname -cred $cred
+
+
+# Default engine ID
+$engineId = 1
+
+# Create Orion Node
+# Update to match your configuration
+$nodeCaption = "my.cbqosnode.corp"
+$newSNMPV2NodeProps = @{
+    IPAddress = "1.1.1.2";
+    EngineID = $engineId;
+    Caption = $nodeCaption;
+    ObjectSubType ='SNMP';
+    Community = "public";
+    SNMPVersion = 2;
+    DNS = "";
+    SysName = "";
+}
```

Comment:
I would remove the unnecessary semicolons from the multi-line hashtable definition.